bin/api.wsgi

```diff
@@ -55,75 +38,64 @@ args.site_name = ' '.join(args.site_name).strip('"\'')
 args.quarantine_path = os.path.join(args.data_path, 'quarantine')
 args.upload_path = os.path.join(args.data_path, 'upload')
 
-app.config = vars(args)
+from api import mongo
+mongo.configure_db(args.db_uri, args.site_id, args.site_name, args.api_uri)
+
+from api.util import log
+from api import api
+from api import centralclient, jobs
+from api import jobs
```

Comment:
Yes, we need to do these imports after mongo e the db handle has been initialized.